homeworks/kirkovvalentiniván/2_vars_datatypes/user_info.py

```diff
@@ -0,0 +1,46 @@
+from pprint import pprint
+
+user_info = {
+    "name": "Mike",
+    "age": 25,
+    "favourite_meals": [
+        "pizza",
+        "carbonara",
+        "sushi"
+    ],
+    "phone_contacts": {
+        "Mary": "+36701234567",
+        "Tim": "+36207654321",
+        "Tim2": "+36304567321",
+        "Jim": "+364005000"
+    }
+}
+
+prog_language = input("List 4 programming languages. Separate them by comma without using space.")  #1a Ask for 4 programming languages
+prog_language_list = prog_language.split(",")                                                       #1b Conversion to lists
+user_info["skills"] = prog_language_list                                                            #1c List adding to dictionaries
+
+user_info["favourite_meals"] = sorted(user_info["favourite_meals"])                                 #2. Sorting to alphabetical order
```

Comment:
a sorted() is jó megoldás, de mivel a dictionary egy mutable object, meghívhatod rá a sort() metódust, és így nem kell reassign-olni. Tehát ehelyett:
user_info["favourite_meals"] = sorted(user_info["favourite_meals"]) 

elég ennyi is:
user_info["favourite_meals"].sort(). 

Nem hiba, csak fyi.